indico/modules/events/editing/controllers/base.py

```diff
@@ -56,4 +56,4 @@ def _user_is_authorized_editor(self):
         if session.user.is_admin:
             # XXX: not sure if we want to keep this, but for now it's useful to have!
             return True
-        return self.editable.editor == session.user
+        return self.event.can_manage(session.user, permission='paper_editing')
```

Comment:
You need to be the assigned editor to submit a review etc. so this isn't correct.